webpack.config.js

```diff
@@ -157,14 +151,15 @@ module.exports = {
 
             // same goes for js-sdk
             "matrix-js-sdk": path.resolve('./node_modules/matrix-js-sdk'),
+
+            "Webapp": path.resolve('./webapp'),
```

Comment:
Out of curiosity... why capital "W" here? The capital seems a bit out of place, since I don't think any other paths will have them... If you want the name to suggest it's a special keyword, maybe prefix the alias with some token like "$webapp"?

Reply:
Mostly just as it seemed to be a semi-convention with other people who used resolve.alias, to distinguish it from a real npm module which are generally lowercase. I assume $ would work too.

Reply:
Ah okay, I see that's the convention described in Webpack docs, for example. Well, whichever seems best to you then.